cmd/zc_traverser_s3.go

```diff
@@ -214,6 +218,10 @@ func newS3Traverser(credentialType common.CredentialType, rawURL *url.URL, ctx c
 			LogError: glcm.Error,
 		})
 
+	if t.logLevel == pipeline.LogDebug {
```

Comment:
azcopyScanningLogger knows what level to log on, logLevel should need not be inherited from s3Client